OpenRA.Mods.Common/Traits/Buildings/Bridge.cs

```diff
@@ -47,7 +47,7 @@ class BridgeInfo : ITraitInfo, IRulesetLoaded, Requires<HealthInfo>, Requires<Bu
 		public WeaponInfo DemolishWeaponInfo { get; private set; }
 
 		[Desc("Types of damage that this bridge causes to units over/in path of it while being destroyed/repaired. Leave empty for no damage types.")]
-		public readonly HashSet<string> DamageTypes = new HashSet<string>();
+		public readonly BitSet<DamageType> DamageTypes = default(BitSet<DamageType>);
```

Comment:
Consistency nit: for `BitSet<TargetableType>` we left the default value implicit by not assigning anything, or in a couple of places using `new BitSet<TargetType>()`.  It would be nice if we could be uniform, probably by changing the uses of `BitSet<TargetType>`.

Reply:
Not assigning anything produces warnings; standardize on default() ?

Reply:
Yes please.